frontend/app/src/pages/main/workers/components/worker-table.tsx

```diff
@@ -107,7 +107,7 @@ export function WorkersTable() {
           Last seen {relativeDate(data?.lastHeartbeatAt)} <br />
           {(data.maxRuns ?? 0) > 0
             ? `${data.availableRuns} / ${data.maxRuns ?? 0}`
-            : '∞'}{' '}
+            : '100'}{' '}
```

Comment:
we default to 100 here?

Reply:
Yeah, this is purely on the backend but is a safeguard for avoiding overloading the workers